docs/features/SERIAL-CONFIG-QUICKREF.md

```diff
@@ -0,0 +1,131 @@
+# Serial Configuration UI - Quick Reference
+
+**Full Specification:** See [docs/features/serial-configuration-ui.md](./serial-configuration-ui.md)
+
+## What This Feature Enables
+
+Users can configure and test radar serial ports through a web interface instead of editing systemd service files.
+
+## Core Requirements
+
+1. **Database Schema** - Store serial port configurations in SQLite
+2. **API Endpoints** - REST API for CRUD operations and testing
+3. **Serial Testing** - Validate port connectivity and auto-detect baud rate
+4. **Web UI** - User-friendly interface at `/settings/serial`
+5. **Server Integration** - Load configs from DB at startup
+6. **Backward Compatible** - CLI flags still work
+
+## Implementation Phases
+
+### Phase 1: Database Foundation (1-2 days)
+- Create `radar_serial_config` table
+- Migration with default HAT configuration
+- Server loads from database
+- CLI flag fallback for compatibility
+
+### Phase 2: API Endpoints (3-4 days)
+- `/api/serial/configs` - CRUD operations
+- `/api/serial/test` - Test connection
+- `/api/serial/detect-baud` - Auto-detect baud rate
+- Unit and integration tests
+
+### Phase 3: Web UI (4-5 days)
+- Configuration list page
+- Edit/create modal
+- Test connection UI
+- Auto-detect button
+- User documentation
+
+### Phase 4: Multi-Sensor (Future)
+- Multiple SerialMux instances
+- Data tagging with sensor ID
+- Multi-sensor analytics
+
+## Key Design Decisions
+
+1. **Database over config files** - Consistent with existing patterns
+2. **Read-only testing** - Safe and non-disruptive
+3. **Selectable baud rates** - Prevents errors, auto-detect as helper
+4. **Multiple SerialMux instances** - Future-ready for multi-sensor
+
+## Database Schema (FR1)
+
+```sql
+CREATE TABLE IF NOT EXISTS radar_serial_config (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name TEXT NOT NULL UNIQUE,
+    port_path TEXT NOT NULL,
+    baud_rate INTEGER NOT NULL DEFAULT 19200,
+    data_bits INTEGER NOT NULL DEFAULT 8,
+    stop_bits INTEGER NOT NULL DEFAULT 1,
+    parity TEXT NOT NULL DEFAULT 'N',
+    enabled INTEGER NOT NULL DEFAULT 1,
+    description TEXT,
+    sensor_model TEXT DEFAULT 'OmniPreSense OPS243-A',
```

Comment:
sensor_model needs to be a join key to a table of valid models. OPS243-A will have dopler and no distance, OPS243-C will have FMCW and Doppler so there is a distance value returned too. this will lead to different startup commands that need to be issued